couchdbkit/ext/django/schema.py

```diff
@@ -75,7 +76,7 @@ def contribute_to_class(self, cls, name):
                 # Ignore any private attributes that Django doesn't care about.
                 # NOTE: We can't modify a dictionary's contents while looping
                 # over it, so we loop over the *original* dictionary instead.
```

Comment:
(fwiw, you can just use `keys()`:

    for name in self.meta.keys():
        if we_dont_want(name):
            del meta_attrs[name]
)